src/main/java/uk/ac/cam/cl/dtg/segue/api/managers/UserAuthenticationManager.java

```diff
@@ -319,6 +323,23 @@ public RegisteredUser getUserFromSession(final HttpServletRequest request) {
             return null;
         }
 
+        if (properties.getProperty(Constants.SEGUE_APP_ENVIRONMENT).equals(EnvironmentType.PROD.name())) {
```

Comment:
I wonder whether checking this once on Manager creation and storing the boolean as a class attribute would be better than checking every request?

Reply:
Possibly, but I think the properties all get preloaded into a map in memory anyway so it is probably not much difference.

Reply:
In which case I will leave it as-is!